src/builders.ts

```diff
@@ -26,7 +26,7 @@ export function buildFromTodoOperations(
   for (const todoOperation of todoOperations) {
     const [operation, todoDatum] = toTodoDatum(todoOperation);
 
-    if (todoDatum.engine !== engine) {
+    if (engine !== 'all' && todoDatum.engine !== engine) {
```

Comment:
Just to confirm, this means we now have "reserved" the engine named `all`, right?

(I think that's 100% fine, just wanted to make sure that is the point/goal here)

Reply:
Yep that's accurate.